lib/proto/annotations_grpc_pb.js

```diff
@@ -1 +1 @@
-// GENERATED CODE -- NO SERVICES IN PROTO
+// GENERATED CODE -- NO SERVICES IN PROTO
```

Comment:
Not sure why we're getting these changes, see if regenerating and committing removes this if not I'll take a closer look.